test/libsolidity/syntaxTests/tupleAssignments/empty_tuple_assignment.sol

```diff
@@ -0,0 +1,12 @@
+function f() {
+    ((, ())) = (1, 3);
+}
+
+function g() {
```

Comment:
```suggestion
function g() public pure {
```

Reply:
I only added pure. Can't add public to free functions.